NewsApp/NewsApp/Model/Article.swift

```diff
@@ -13,8 +13,20 @@ struct Article: Codable {
     let author: String?
     let title: String?
     let articleDescription: String?
-    let url: URL?
-    let urlToImage: URL?
+    private let urlString: String?
+    var urlWeb: URL? {
+        if let urlString = urlString {
+             return URL(string: urlString)
+        }
+        return nil
+    }
+    private let urlImageString: String?
+    var urlImage: URL? {
```

Comment:
le type plutôt en suffixe pour la convention imageURL, webURL

Reply:
fix in "fix PR"